dvc/commands/experiments/init.py

```diff
@@ -57,7 +57,6 @@ def run(self):
                 "metrics": self.args.metrics,
                 "params": self.args.params,
                 "plots": self.args.plots,
-                "live": self.args.live,
```

Comment:
This is still documented, and I think it writes out `metrics` and `plots` keys now instead of a `live` section, so I think it's probably better for now to keep it and update its paths.

Reply:
Okay, I added it back and see now that you did update the paths in this PR, so LGTM 👍